src/thematic/1_water/sfp-106-nl_waterpeil-peilbuis/04_principe.Rmd

```diff
@@ -0,0 +1,45 @@
+# Principe
+
+```{=html}
+<!--
+Korte omschrijving van de principes waarop het protocol gebaseerd is.
+Bv principe achter het nemen van mengmonsters, principe achter visueel inschatten van de bedekking van een plant, ...
+-->
+```
+## Manuele peilmeting
+
+De peilmeting is de afstand tussen het referentiepunt (de bovenrand van de peilbuis zonder dop) en het wateroppervlak in de peilbuis (figuur \@ref(fig:Figuur1)).
+Omdat deze meting naar beneden gericht is wordt dit met een negatief cijfer genoteerd.
+Komt bij een overstroming het peil boven het meetpunt, dan moet een positieve waarde genoteerd worden.
```

Comment:
"meetpunt" is onduidelijk. Bedoel je hiermee de top van de peilbuis?

Reply:
Inderdaad, dit moet 'referentiepunt' zijn. De zin is aangepast om dit op te helderen